frontend/src/components/maps/leaflet/LayerPopup/constants/index.tsx

```diff
@@ -32,6 +35,32 @@ export const parcelLayerPopupConfig = {
       </>
     ),
   },
+  // GEN_GROSS_LAND_VALUE: {
+  //   label: 'Gross Land Value:',
+  //   display: (data: any) => (
+  //     <>{data.GEN_GROSS_LAND_VALUE ? `$${data.GEN_GROSS_LAND_VALUE}` : `N/A`}</>
+  //   ),
+  // },
+  // GEN_GROSS_IMPROVEMENT_VALUE: {
+  //   label: 'Gross Improvement Value:',
+  //   display: (data: any) => (
+  //     <>{data.GEN_GROSS_IMPROVEMENT_VALUE ? `$${data.GEN_GROSS_IMPROVEMENT_VALUE}` : `N/A`}</>
+  //   ),
+  // },
+  // GEN_NET_LAND_VALUE: {
+  //   label: 'Net Land Value:',
+  //   display: (data: any) => <>{data.GEN_NET_LAND_VALUE ? `$${data.GEN_NET_LAND_VALUE}` : `N/A`}</>,
+  // },
+  // GEN_NET_IMPROVEMENT_VALUE: {
+  //   label: 'Net Improvement Value:',
+  //   display: (data: any) => (
+  //     <>{data.GEN_NET_IMPROVEMENT_VALUE ? `$${data.GEN_NET_IMPROVEMENT_VALUE}` : `N/A`}</>
+  //   ),
+  // },
+  // FOLIO_ID: {
+  //   label: 'Folio ID:',
+  //   display: (data: any) => <>{data.FOLIO_ID ? `${data.FOLIO_ID}` : `N/A`}</>,
+  // },
```

Comment:
I'm good with leaving this here. Easy to uncomment if we want it back.

Reply:
Yeah I felt there was a non-zero chance we might move the information around so I figured I would leave it there for now.